src/test/java/pl.put.poznan.builder/ElementTest.java

```diff
@@ -0,0 +1,33 @@
+
```

Comment:
Could you please add package statements ( `package pl.put.poznan.builder;`) as the first lines of those new files? I see that IntelliJ is complaining about it.

Reply:
Could you check, if it's what you mean?